src/main/java/org/jenkinsci/plugins/parallel_test_executor/ParallelTestExecutor.java

```diff
@@ -312,4 +341,11 @@ public String getDisplayName() {
             return "Parallel test job execution";
         }
     }
+
+    static Predicate<FlowNode> stageNamePredicate(@Nonnull String stageName) {
+        // Arrays.asList in to prevent compiler warning due to unchecked generics/varargs whackiness.
```

Comment:
Someone not yet using `@SafeVarargs`?

Reply:
`Predicates.and`, yup. Ancient Guava fun.